src/app.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// Write your code here
+import readline from 'readline';
+import { generateRandomNumber } from './modules/generateRandomNumber';
+import { checkIsValidUserInput } from './modules/checkIsValidUserInput';
+import { getBullsAndCows } from './modules/getBullsAndCows';
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const numberToGuess = generateRandomNumber();
+
+terminal.question('Guess the number', (userNumber) => {
+  if (!checkIsValidUserInput(userNumber)) {
+    return;
+  }
+
+  if (userNumber === numberToGuess) {
+    terminal.close();
+  } else {
+    getBullsAndCows(userNumber, numberToGuess);
```

Comment:
The result of getBullsAndCows is not being used. Consider storing the result and providing feedback to the user about the number of bulls and cows.